cli/tests/test_workspaces.rs

```diff
@@ -944,11 +941,11 @@ fn test_workspaces_current_op_discarded_by_other(automatic: bool) {
     insta::allow_duplicates! {
         insta::assert_snapshot!(output, @r"
         Working copy changes:
-        A added
+        C {modified => added}
```

Comment:
Is this expected?

Reply:
Yes, it's because the Git backend supports detecting copies (the local backend does)